src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -250,11 +250,37 @@ public static class TextCatalog
         /// <param name="catalog">The text-related transform's catalog.</param>
         /// <param name="columns">Pairs of columns to remove stop words on.</param>
         /// <param name="language">Langauge of the input text columns <paramref name="columns"/>.</param>
-        public static StopWordsRemovingEstimator RemoveStopWords(this TransformsCatalog.TextTransforms catalog,
+        public static StopWordsRemovingEstimator RemoveDefaultStopWords(this TransformsCatalog.TextTransforms catalog,
             (string input, string output)[] columns,
              StopWordsRemovingEstimator.Language language = StopWordsRemovingEstimator.Language.English)
             => new StopWordsRemovingEstimator(Contracts.CheckRef(catalog, nameof(catalog)).GetEnvironment(), columns, language);
 
+        /// <summary>
+        /// Removes stop words from incoming token streams in <paramref name="inputColumn"/>
+        /// and outputs the token streams without stopwords as <paramref name="outputColumn"/>.
+        /// </summary>
+        /// <param name="catalog">The text-related transform's catalog.</param>
+        /// <param name="inputColumn">The column containing text to remove stop words on.</param>
+        /// <param name="outputColumn">The column containing output text. Null means <paramref name="inputColumn"/> is replaced.</param>
+        /// <param name="stopwords">Array of words to remove.</param>
```

Comment:
>// <param name="stopwords">Array of words to remove.</param> [](start = 9, length = 60)

did you want to reference the sample from here?

Reply:
Am I doing this right?

---
In reply to: [250825805](https://github.com/dotnet/machinelearning/pull/2236#discussion_r250825805) [](ancestors = 250825805)